src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessExitListener.java

```diff
@@ -19,8 +19,10 @@
 /**
  * Default process exit listener that simply captures the exit code and makes it available with a
  * getter. Before the process exit code is captured, it's initialized to -1.
+ *
+ * <p>When used with {@link com.google.cloud.tools.app.impl.cloudsdk.internal.sdk.CloudSdk.Builder},
+ * this allows checking of the success or failure of an action execution.
```

Comment:
I'd get rid of this until you feel real pain from the lack of it.  With
Java 8 it will be even less necessary.

On Fri, Jun 3, 2016 at 1:17 PM, meltsufin <notifications@github.com> wrote:

> In
> src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/process/DefaultProcessExitListener.java
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/100#discussion_r65743599>
> :
>
> > @@ -19,8 +19,10 @@
> >  /**
> >   * Default process exit listener that simply captures the exit code and makes it available with a
> >   * getter. Before the process exit code is captured, it's initialized to -1.
> > + *
> > + * <p>When used with {@link com.google.cloud.tools.app.impl.cloudsdk.internal.sdk.CloudSdk.Builder},
> > + * this allows checking of the success or failure of an action execution.
>
> I'm not really sure, but you save a few lines of code and the cost of the
> anonymous classes.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GoogleCloudPlatform/app-tools-lib-for-java/pull/100/files/74bc029717648db8bcb7400400a7026cdd5a8ec0..3f8439b8de16e3680350edb80b6d94cdc49d5d4e#r65743599>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AHf5HWYxxZHlcRFnkdNxaEXFo4ouTVnVks5qIGG5gaJpZM4Itp2k>
> .
>